### DISCUSSION TITLE
How to work with external screen and keep mouse control?

### DISCUSSION THREAD

**Discussion by akrz91** [blinksh/blink#1578] [Support] [answered]
Hi,

I’d like to work on blink shell and VS code on the external screen so that I might use bigger workspace.
I’m able to use the mouse controller only on the embeded iPad screen.
The Blink shell automatically detects external screen attached and uses it as another output device with new command shell. I am able to focus on that screen with keyboard shortcut cmd+O but it seems like there is only keyboard support - no mouse control.

So there is no way of using VS code on the bigger screen as the external screen is not supporting the mouse controller and is not even able to mirror the screen from the iPad since blink shell by default displays another new console on the external screen.

Please correct me if I’m wrong :)
If that’s the way it should work so far, I’d like to know if it will be fixed in the future?

**Comment by carloscabanero** [comment, marked as answer]
Hi! This is a limitation of iPadOS that will be fixed with Stage Manager on 16. Moving to discussions.
